docs/app/pages/components/sections/panels/item-display-panel-inline/item-display-panel-inline.component.html

```diff
@@ -3,7 +3,7 @@
 <div class="demo-attributes item-display-panel-inline-container">
     
     <div class="table-responsive preview-pane-table-demo">
-        <table class="table table-hover item-display-panel-inline" [class.animate]="animate" [class.item-display-panel-visible]="visible">
+        <table class="table table-hover item-display-panel-inline animate" [class.item-display-panel-visible]="visible">
```

Comment:
Seem to have a bottom border on the panel, looks a bit off as the table extends further than it. There is also a border left that only comes part of the way down also:

![image](https://user-images.githubusercontent.com/20795331/28065899-907f100e-6632-11e7-8fcc-1e1aafe87717.png)